packages/flutter/test/material/dropdown_test.dart

```diff
@@ -380,11 +386,15 @@ void main() {
 
   testWidgets('Dropdown button with no app', (WidgetTester tester) async {
     String value = 'one';
+    StateSetter setState;
     void didChangeValue(String newValue) {
-      value = newValue;
+      setState(() {
+        value = newValue;
+      });
     }
 
-    Widget build() {
+    Widget build() => StatefulBuilder(builder: (BuildContext context, StateSetter setter) {
```

Comment:
our style guide suggest we only use arrow function if content fit into a single line.